cylc/flow/etc/job.sh

```diff
@@ -36,16 +36,6 @@ cylc__job__main() {
         set -x
     fi
     # Prelude
```

Comment:
Can remove the `# Prelude` comment line as well.

Reply:
Done.